lib/models/package_wrapper.dart

```diff
@@ -61,17 +61,22 @@ class Package with _$Package {
       name: 'packageType',
       unknownEnumValue: PackageType.unknown,
     )
-        PackageType packageType,
+    PackageType packageType,
 
     /// StoreProduct assigned to this package.
     // ignore: invalid_annotation_target
-    @JsonKey(name: 'product')
-        StoreProduct storeProduct,
+    @JsonKey(name: 'product') StoreProduct storeProduct,
 
-    /// Offering this package belongs to.
-    String offeringIdentifier,
+    /// Offering context this package belongs to.
+    PresentedOfferingContext presentedOfferingContext,
   ) = _Package;
 
   factory Package.fromJson(Map<String, dynamic> json) =>
       _$PackageFromJson(json);
 }
+
+extension ExtendedPackage on Package {
+  /// Offering this package belongs to.
+  @Deprecated('use presentedOfferingContext')
+  String get offeringIdentifier => presentedOfferingContext.offeringIdentifier;
+}
```

Comment:
Adding a getting property in an extension to prevent a breaking change. `fromJson` breaks if this getting property is directly on the class.